modules/person.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+
+class Person(object):
+    def __init__(self, **kwargs):
+        pass
+
+# Adds a person to the system and allocates the person to a random room.
+    def add_person(self, emp_id, name):
+        self.file_name = open(os.path.dirname(__file__), 'amity_load.txt', mode='w').read()
+        self.file_name.close()
+        self.emp_id = []
+        self.name = []
+        self.add_person({self.emp_id: self.name})
```

Comment:
Doesn't this function call itself recursively? Especially without the correct num of args. It'll most likely crush